src/Symfony/Component/Validator/Constraints/WordCountValidator.php

```diff
@@ -44,7 +44,7 @@ public function validate(mixed $value, Constraint $constraint): void
         $words = iterator_to_array($iterator->getPartsIterator());
 
         // erase "blank words" and don't count them as words
-        $wordsCount = \count(array_filter(array_map(trim(...), $words)));
+        $wordsCount = \count(array_filter(array_map(trim(...), $words), 'mb_strlen'));
```

Comment:
```suggestion
        $wordsCount = \count(array_filter(array_map(trim(...), $words), fn ($word) => '' !== $word));
```

What about this to not depend on the mbstring extension?

Reply:
makes sense @xabbuh. Applied